sidecar/Dockerfile

```diff
@@ -33,9 +35,11 @@ RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2
    rm -f awscliv2.zip
 
 COPY sidecar/run-sidecar.sh /usr/local/bin
+COPY sidecar/sidecar-script.py /usr/local/bin
 
 RUN cd /root
 
 # run it forever
-CMD ["/usr/local/bin/run-sidecar.sh"]
+# CMD ["/usr/local/bin/run-sidecar.sh"]
```

Comment:
Please put a comment here that we are using one script here as a default but it can change to another one.